libass/ass_font.c

```diff
@@ -579,16 +586,18 @@ static void ass_glyph_embolden(FT_GlyphSlot slot)
 /**
  * Slightly italicize a glyph
  */
-static void ass_glyph_italicize(FT_GlyphSlot slot)
+static void ass_glyph_italicize(FT_Face face)
 {
     FT_Matrix xfrm = {
         .xx = 0x10000L,
         .yx = 0x00000L,
-        .xy = 0x05700L,
+        .xy = ass_face_is_postscript(face)
+            ? 0x02d24L /* tan(10 deg) */
+            : 0x05700L /* from GDI; roughly 0.34; tan(18.77 deg) */,
```

Comment:
Technically, it is more ``sin(20 deg)`` instead of ``tan(18.77 deg)``

Reply:
Thanks for the pointer. However, sin has no physical meaning in this context, whereas tan refers to the slant angle. I can add a comment that it may have apparently originated as sin(20°), but it doesn’t seem very useful—nor is it even a very good approximation of sin(20°), although certainly better than of tan(20°) for what it’s worth.

Reply:
It's sin(20°)*256, truncated to an integer, and then shifted left 8 to expand to 8.8 form.

Reply:
Well, yeah. “Truncated to 1/8th” being “not a very good approximation” (even _rounding_ to 1/8th would have produced a different value). Anyway, is this relevant enough that you want it mentioned?

Reply:
Reworded to add a mention of sin(20°). Does this look good?

Reply:
Aaand removed again and reworded some more to avoid unofficial data.